xml/admin_logging.xml

```diff
@@ -0,0 +1,446 @@
+    <listitem>
+     <para>
+      quiet
+     </para>
+    </listitem>
+   </itemizedlist>
+
+   <para>
+   For detailed explanations of the usage of these log levels please see:
+   <link xlink:href="https://docs.saltstack.com/en/latest/ref/configuration/logging/">Salt Log Levels (Upstream)</link>
+   </para>
+
+   <sect3 xml:id="sec.admin.logging.log-levels.salt.debug">
+    <title>Setting a different log level</title>
+    <para>
+    When debugging issues, <literal>debug</literal> is usually enough and is
+    what the &productname; engineers use to debug problems.
+    </para>
+
+    <sect4 xml:id="sec.admin.logging.log-levels.salt.debug.master">
+     <title>Salt Master</title>
+
+    <para>
+     In the case of the <literal>salt-master</literal>, this configuration can be
+     modified in the admin node, at <filename>/etc/caasp/salt-master-custom.conf</filename>.
+     Inside this file you can add: <literal>log_level: debug</literal>.
+    </para>
+
+    <para>
+     Note that after any change on this file you need to restart the
+     <literal>salt-master</literal> container, like:
+    </para>
+<screen>
+<command>docker rm -f $(docker ps | grep salt-master | awk '{print $1}')</command>.
+     </screen>
+
+    <para>
+     After deleting this container, the <literal>kubelet</literal> will bring a
+     new <literal>salt-master</literal> container automatically with the new
+     configuration applied. So you can check the logs with the
+     <literal>debug</literal> loglevel.
+     </para>
+
+<screen>
+<command>docker logs -f $(docker ps | grep salt-master | awk '{print $1}')</command>
+   </screen>
+  </sect4>
+
+  <sect4 xml:id="sec.admin.logging.log-levels.salt.debug.minion">
+   <title>Salt Minions</title>
+   <note>
+    <title>Modifications on Salt minions are not persistent</title>
+    <para>
+     These changes are discarded when the node is updated with a new snapshot by
+     <literal>transactional-update</literal>.
+    </para>
+  </note>
+   <para>
+    In the case of the <literal>Salt</literal> minions, there's no "official"
+    way to tweak this configuration, however, a new file can be added on the
+    minion that you want to print debugging <literal>Salt</literal> information
+    in: <filename>/etc/salt/minion.d/100-debug.conf</filename>. After doing a
+    <command>systemctl restart salt-minion</command> you can check that it now
+    prints debugging information with <command>journalctl -fu salt-minion</command>.
+   </para>
+
+  </sect4>
+  </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.logging.log-levels.kubernetes">
+   <title>&kube; Log Levels</title>
+   <para>
+    For &kube; our default <literal>loglevel</literal> is <literal>2</literal>
+    <link xlink:href="https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging">Kubernetes Upstream: Output Verbosity and Debugging</link>.
+    However, this setting can be tweaked by using pillar overrides. If modified,
```

Comment:
We should explain step by step how this is done, and again probably remove salt slang from here (`pillar override`).

Reply:
@ereslibre Please provide me with the "step by step"-steps

Reply:
Will do.

Reply:
For this to be valid we need to apply https://github.com/kubic-project/velum/pull/640 (fixing bsc#1106193)

Reply:
@ereslibre Can you put how to enable verbosity on a cluster as discussed?

Reply:
Once https://github.com/kubic-project/velum/pull/640 is merged, they can do the following:

* If it's before the cluster first formation
  * Run this command on the admin node anytime before clicking on "Bootstrap cluster"
    * `docker exec -it $(docker ps | grep dashboard | awk '{print $1}') entrypoint.sh bundle exec rake "velum:create_pillar['kube_log_level', 'desired_loglevel']"`

* If the cluster is already formed, and you want to globally modify the loglevel of all Kubernetes components in all machines
  * Run these commands on the admin node:
    * `docker exec -it $(docker ps | grep dashboard | awk '{print $1}') entrypoint.sh bundle exec rake "velum:create_pillar['kube_log_level', 'desired_loglevel']"`
    * `docker exec -it $(docker ps | grep salt-master | awk '{print $1}') salt-run state.orchestrate orch.kubernetes`

In both cases, `desired_loglevel` refer to [a number in the documented Kubernetes verbosity levels](https://github.com/kubic-project/salt/blob/3dca76e2b679a53e80b4dc99634985bd78cebc40/pillar/params.sls#L60-L69).

Example:

 * `docker exec -it $(docker ps | grep dashboard | awk '{print $1}') entrypoint.sh bundle exec rake "velum:create_pillar['kube_log_level', '7']"`

NOTE: For this verbosity level to actually apply in the cluster, https://github.com/kubic-project/velum/pull/640 needs to get merged.